app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
I think the greater value is in communicating clearly. 

We're not trying to identify some technology that's auto-generated something - we're leaving a comment for visitors to read.